codalab/apps/web/models.py

```diff
@@ -287,6 +287,13 @@ class Competition(ChaHubSaveMixin, models.Model):
     hide_top_three = models.BooleanField(default=False, verbose_name="Hide Top Three Leaderboard")
     hide_chart = models.BooleanField(default=False, verbose_name="Hide Chart")
     allow_organizer_teams = models.BooleanField(default=False, verbose_name="Allow Organizer Teams")
+    upper_bound_max_submission_size = models.PositiveIntegerField(
+        default=300,
+        validators=[
+            MinValueValidator(0)
```

Comment:
The organizer can only set the max submission size to 0. It will work the same way as any other value. It does not seem to have any side effect from what I've tested